integreat_cms/core/management/commands/repair_tree.py

```diff
@@ -0,0 +1,88 @@
+import logging
+
+from django.core.management.base import CommandError
+
+from ....cms.models import Page
+from ..log_command import LogCommand
+
+logger = logging.getLogger(__name__)
+
+
+class Command(LogCommand):
+    """
+    Management command to repair a broken Treebeard page tree
+    """
+
+    help = "Search & replace links in the content"
+
+    def add_arguments(self, parser):
+        """
+        Define the arguments of this command
+
+        :param parser: The argument parser
+        :type parser: ~django.core.management.base.CommandParser
+        """
+        parser.add_argument(
+            "--page", help="The ID of a page belongig to the broken tree."
```

Comment:
```suggestion
            "page_id", type=int, help="The ID of a page belonging to the broken tree."
```
Command line options starting with `--` are usually optional, but in this case we want the id to be obligatory, right? If we remove the dashes, the parser will require this argument.